README.md

```diff
@@ -26,6 +26,7 @@ Keybinding                              | Description
 <kbd>Ctrl+K</kbd>, <kbd>Shift+Tab</kbd> | Previous view.
 <kbd>Ctlr+J</kbd>, <kbd>Tab</kbd>       | Next view.
 <kbd>Ctrl+H</kbd>, <kbd>Alt+H</kbd>     | Toggle history.
+<kbd>Ctrl+E</kbd>                       | Export as cURL command
```

Comment:
I don't know about this keybinding - it's the "end of line" keybind in readline, which is common in a lot of applications (especially TUI applications) on *nix systems, and which might be worth implementing in wuzz eventually. The eventual solution of course should be customizable keybinds, as @asciimoo has mentioned before, but for now I would suggest something like `ctrl+u`.

It might be that my concerns are unwarranted - I'd appreciate anybody else's thoughts on this (@asciimoo?).